extensions/vscode/src/utils/guid.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (C) 2024 by Posit Software, PBC.
+
+export const GuidRegex =
```

Comment:
Nitpicky: Typically I see these types of constants named like `GUID_REGEX`

Reply:
updated.